client/algod/wrappers.go

```diff
@@ -4,6 +4,7 @@ import (
 	"context"
 	"errors"
 	"fmt"
+	"strings"
```

Comment:
Can you please sort imports alphabetically? With the `go-algorand-sdk` imports coming last. Thank you!

Reply:
Sure! Done in latest commit.